docs/eventlisteners.md

```diff
@@ -945,3 +950,55 @@ Except as otherwise noted, the content of this page is licensed under the
 [Creative Commons Attribution 4.0 License](https://creativecommons.org/licenses/by/4.0/),
 and code samples are licensed under the
 [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
+
+## EventListener Secure Connection
+
+Triggers now support both `HTTP` and `HTTPS` connection by adding some configurations in eventlistener. 
+### Prerequisites
+* Certficates.
+* Secret which includes those certificates.
+
+There are two env var reserved for `HTTPS` connection to read certificates
+1. **SSL_CERT_FILE**: specify the location of mounted cert file.
+2. **SSL_KEY_FILE**: specify the location of mounted key file.
+
+As mentioned in the prerequisites users are asked to create their own certificates, secrets and mount as a volume in the 
+`resources` section of eventlistener as shown below.
+
```

Comment:
We definitely don't have to do this in this PR but it would be really nice to have an example/guide that runs through the whole flow i.e. creating certs, adding them to the secret, modifying the EL etc.

Reply:
That's really good idea
Modified docs and added README to examples.